client/src/components/BulkCreateForm/BulkCreateForm.js

```diff
@@ -516,61 +553,83 @@ export default class BulkCreateForm extends React.Component {
                 <div className="validation-table-wrapper mt-4 mb-1">
                     <table className="table validation-table">
                         <thead>
-                            <tr>
-                                <th>#</th>
-                                <th>Warnings</th>
-                                <th>Errors</th>
-                            </tr>
+                        <tr>
+                            <th>#</th>
+                            <th>Title</th>
+                            <th>Artist</th>
+                            <th>Address</th>
+                            <th>Date Created</th>
+                            <th>Warnings</th>
+                            <th>Errors</th>
+                        </tr>
                         </thead>
                         <tbody>
-                            {
-                                results.map(result => (
-                                    <tr key={result.index}>
-                                        <td>{result.index}</td>
-                                        <td>{result.warnings.map((warning, index) => (
-                                            <div key={index} dangerouslySetInnerHTML={{__html: warning}} className="bulk-warning" />
-                                        ))}</td>
-                                        <td>{result.errors.map((error, index) => (
-                                            <div key={index} className="bulk-warning">{error}</div>
-                                        ))}</td>
-                                    </tr>
-                                ))
-                            }
+                        {results.map(result => {
+                            let suggestion = result['monumentSuggestion']
+
+                            return (
+                                <tr key={result.index}>
+                                    <td>{result.index}</td>
+                                    <td>{suggestion.title}</td>
+                                    <td>{suggestion.artist}</td>
+                                    <td>{suggestion.address}</td>
+                                    <td>{suggestion.date.split('T')[0]}</td>
```

Comment:
This will crash if one of the monuments a user is uploading doesn't have a value in the date field. Just need to add a check to see if it's null. 
![image](https://user-images.githubusercontent.com/35973124/137196153-c1d7bd46-1578-416d-8a00-9b9e97658917.png)

Reply:
Good catch, I gave it a default for when there's no value